rustfmt.toml

```diff
@@ -1 +1,2 @@
 max_width = 80
+edition = "2018"
```

Comment:
I don't think this is needed. It'll read the cargo file.

Reply:
So it will be read from Cargo.toml if you use `cargo fmt`, but not if you run `rustfmt` directly. If we want to standardize around using cargo fmt I can update the Makefile to use it instead of rustfmt, but otherwise I think we want to leave this.

Reply:
I think standardising everything to use cargo would be a good idea 😃

Reply:
K, updated the Makefile and removed this line